state/protocol/prg/customizers.go

```diff
@@ -16,14 +16,18 @@ import (
 
 var (
 	// ConsensusLeaderSelection is the customizer for consensus leader selection
-	ConsensusLeaderSelection = customizerFromIndices(0, 1, 1)
+	ConsensusLeaderSelection = customizerFromIndices(0)
```

Comment:
Why did these lines change?

Reply:
I have to defer to @tarakby, this commit was added by him I think.

Reply:
This specific line update has nothing to do with this new feature. 
What is required is to add a new `ExecutionRandomSourceHistory` customizer. I noticed the customizer values were messed up so I took the same opportunity to tidy them up. 
(https://github.com/onflow/flow-go/commit/86aed887c3fc13a6a3873c2b52021daa45e8d383). 

I'll add this to the PR description.